### ISSUE TITLE
Support for peripheral derivedFrom

### ISSUE THREAD

**Issue by marcohald** [antoniovazquezblanco/SVD-Parser#20]
First of all thanks for your great ghidra Plugin.
It would be really helpful if you could support peripheral derivedFrom
I searched the Repo and found this Key in you example SVD https://github.com/antoniovazquezblanco/SVD-Parser/blob/6ecbe54bdc58ed3fb7f12b17bf37daedf6c7c0ea/src/test/resources/00_sample.svd#L767
My use Case would be the GPIOC of an STM32L0x2.svd
which is just referenced as:
```
      <peripheral derivedFrom="GPIOB">
         <name>GPIOC</name>
         <baseAddress>0x50000800</baseAddress>
      </peripheral>
```
I attached  my file here 
[STM32L0x2.svd.zip](https://github.com/user-attachments/files/18118672/STM32L0x2.svd.zip)

**Comment by antoniovazquezblanco** [comment]
Hi @marcohald, hanks for the report.

I had this feature in mind but did not have time to implement it.

I will try to implement it when I find some time to do so but I warn you it will have to be after the holidays. I would like to encourage you to try and open a PR if you are willing. Otherwise, I will try to go over this when I find a slot.

**Comment by antoniovazquezblanco** [comment]
Found an small slot to implement this... Will be available in v0.0.6. :)
